### ISSUE TITLE
Global variables are not updated in the grid

### ISSUE THREAD

**Issue by rysep2** [DevExpress/Blazor#94]
I use the latest stable DevExpress version 19.2.3. When a global variable changes which is also used in the grid, the grid is not updated even after calling `StateHasChanged()`. Only after selecting a row, the specific row gets updated.

I included a sample app:
[BlazorApp2.zip](https://github.com/DevExpress/Blazor/files/4245397/BlazorApp2.zip)

**Comment by zabelin-vladimir** [comment]
Hello @rysep2

We changed DxDataGrid code between versions 19.1 and 19.2. In the previous version, when the *StateHasChanged* method is called, the whole DxDataGrid markup was re-rendered. We tried to optimize this behavior.
You can achieve the necessary behavior in DxDataGrid in version 19.2.3 by calling DxDataGrid's [Refresh](https://docs.devexpress.com/Blazor/DevExpress.Blazor.Base.DxDataGridBase-1.Refresh) method after changing the *ExampleBool* property value. This method reloads data and redraws the whole DxDataGrid component.

**Comment by rysep2** [comment]
Ok, that makes sense.

Working with Remote Data it would be nice to have an additional function that only refreshes the grid (for cases like the example I sent) without getting the data from the server again.

**Comment by zabelin-vladimir** [comment]
Hello @rysep2

Thank you for sharing your thoughts with me. Would you please describe a real-life scenario where it's necessary to re-draw a grid without re-binding it to its data source?

**Comment by rysep2** [comment]
> Thank you for sharing your thoughts with me. Would you please describe a real-life scenario where it's necessary to re-draw a grid without re-binding it to its data source?

For example when I want to change the layout. Let's say I have a column which shows both the name of a book and its author separated by a line break. If I want to switch it to first author and then the name of the book, I have to reload the grid and the data although the data is already there.

I found a workaround for this. Creating both DxDataGridColumns and always show one and hide the other instead of having an if/else in the DisplayTemplate of the DxDataGridColumn. Still it would be nice to have a "refresh UI" function but it's not first priority for me.

**Comment by zabelin-vladimir** [comment]
Hello @rysep2 

Thank you for your clarification. 
In my opinion, your approach with changing the visibility of two different columns is appropriate for your task. Thank you for keeping me informed.
